app/controllers/events_controller.rb

```diff
@@ -0,0 +1,34 @@
+class EventsController < ApplicationController
+  def new
+    @event = Event.new
+  end
+
+  def create
+    @event = Event.new(event_params)
+    if @event.save
+      flash[:success] = 'Event created.'
+      redirect_to current_user
+    else
+      render 'new'
+    end
+  end
+
+  def show
+    @event = Event.find_by(id: params[:id])
+    @attendees = @event.attendees
+    redirect_to root_url and return unless logged_in?
+  end
+
+  def index
+    @events = Event.all
+    # @previous_events = Event.previous
+    # @upcoming_events = Event.upcoming
```

Comment:
[Please fix] Remove commented code.

Reply:
thank you for the review, done  and I hope this is it for all .